src/diskpool/azext_diskpool/manual/_transformers.py

```diff
@@ -35,8 +37,56 @@ def transform_disk_pool_list_output(result):
     usable by the CLI and tools such as jpterm. """
     return build_table_output(result, [
         ('Name', 'name'),
-        ('Availability Zones', 'availabilityZones'),
+        ('Resource Group', 'resourceGroup'),
```

Comment:
May I ask why do we need to modify those value?

Reply:
as service request

Reply:
OK